File: move_section.mjs
import fs from 'fs';

const filePath = '/Users/user/testbasedweb/test-based-nutrition-ui/src/pages/treatments/SportsPerformance.tsx';
let content = fs.readFileSync(filePath, 'utf-8');

// The section we want to move starts with "{/* SECTION 3 — EXPLORE YOUR PATHWAY */}"
const startMarker = "{/* SECTION 3 — EXPLORE YOUR PATHWAY */}";
const endMarker = "        {/* SECTION 7.5 — THE SCIENCE BEHIND PERFORMANCE */}";

const startIndex = content.indexOf(startMarker);
const endIndex = content.indexOf(endMarker, startIndex);

if (startIndex === -1 || endIndex === -1) {
  console.error("Markers not found");
  process.exit(1);
}

// Extract the chunk to move
const chunk = content.substring(startIndex, endIndex);

// Remove the chunk from its original location
content = content.substring(0, startIndex) + content.substring(endIndex);

// Find the insertion point
const insertMarker = "{/* SECTION 2 — SPECIALIST LEADS */}";
const insertIndex = content.indexOf(insertMarker);

if (insertIndex === -1) {
  console.error("Insert marker not found");
  process.exit(1);
}

// Update the chunk's title to SECTION 2 and the specialist's to SECTION 3
let updatedChunk = chunk.replace("{/* SECTION 3 — EXPLORE YOUR PATHWAY */}", "{/* SECTION 2 — EXPLORE YOUR PATHWAY */}\n        <div className=\"pt-8 lg:pt-10\"></div>");
let insertContentRest = content.substring(insertIndex).replace("{/* SECTION 2 — SPECIALIST LEADS */}", "{/* SECTION 3 — SPECIALIST LEADS */}");

// Insert the chunk
const finalContent = content.substring(0, insertIndex) + updatedChunk + insertContentRest;

fs.writeFileSync(filePath, finalContent);
console.log("Move successful");

File: move_section.py
import sys

file_path = '/Users/user/testbasedweb/test-based-nutrition-ui/src/pages/treatments/SportsPerformance.tsx'
with open(file_path, 'r') as f:
    content = f.read()

start_marker = "{/* SECTION 3 — EXPLORE YOUR PATHWAY */}"
end_marker = "        {/* SECTION 7.5 — THE SCIENCE BEHIND PERFORMANCE */}"

start_index = content.find(start_marker)
end_index = content.find(end_marker, start_index)

if start_index == -1 or end_index == -1:
    print("Markers not found")
    sys.exit(1)

chunk = content[start_index:end_index]
content = content[:start_index] + content[end_index:]

insert_marker = "{/* SECTION 2 — SPECIALIST LEADS */}"
insert_index = content.find(insert_marker)

if insert_index == -1:
    print("Insert marker not found")
    sys.exit(1)

chunk = chunk.replace("{/* SECTION 3 — EXPLORE YOUR PATHWAY */}", "{/* SECTION 2 — EXPLORE YOUR PATHWAY */}\n        <div className=\"pt-8 lg:pt-10\"></div>")
content = content[:insert_index] + chunk + content[insert_index:].replace("{/* SECTION 2 — SPECIALIST LEADS */}", "{/* SECTION 3 — SPECIALIST LEADS */}")

with open(file_path, 'w') as f:
    f.write(content)

print("Move successful")
